src/components/Property/TenureDetails.js

```diff
@@ -20,7 +20,11 @@ const TenureDetails = ({
       <ul className="lbh-list hackney-property-alerts">
         <Tenure tenure={tenure} canRaiseRepair={canRaiseRepair} />
         <Alerts alerts={locationAlerts} alertType="Address" />
-        <TenureDetail text="TMO" detail={tmoName} />
+        {tmoName == 'London Borough of Hackney' ? (
+          ''
+        ) : (
+          <TenureDetail text="TMO" detail={tmoName} />
+        )}
```

Comment:
Please could we extract the string into a const like `TMO_HACKNEY_DEFAULT` and put it at the top of the component with a comment like 'Properties with TMO names set to this value aren't actually TMOs'. This context should reduce confusion for a future visitor.

It's also probably best for readability to shorten the logic by using an `&&` instead of a ternary because then we don't need to return the empty string when we want nothing to render.

Pseudocode

```
{tmoName !== TMO_HACKNEY_DEFAULT && ... }
```

Reply:
haha, I was actually thinking about using ternary but then decided to go with the classic one :D 
will update everything now ;)